### ISSUE TITLE
Auto-fixer for type-literal-delimiter?

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [palantir/tslint#3959]
### Bug Report

- __TSLint version__: 5.10
- __TypeScript version__: 2.9.1
- __Running TSLint via__: (pick one) VSCode

#### TypeScript code being linted

```ts
type Foo = { a: number, b: string };
```

with `tslint.json` configuration:

```json
"rules": {
    "type-literal-delimiter": true
}
```

It'd be nice to have an auto-fixer here. Seems trivial?

**Comment by giladgray** [comment]
@JoshuaKGoldberg go for it!

**Comment by giladgray** [comment]
fixed in #3964
